ql/ql/src/queries/style/DeadCode.ql

```diff
@@ -12,4 +12,4 @@ import codeql_ql.style.DeadCodeQuery
 
 from AstNode node
 where isDead(node)
-select node, "Code is dead"
+select node, "Code is dead."
```

Comment:
While I appreciate the nihilism expressed by this statement, perhaps "This code is never used." would be more in line with our other alert messages.